src/pybind/mgr/snap_schedule/module.py

```diff
@@ -75,6 +78,8 @@ def snap_schedule_get(self,
         List current snapshot schedules
         '''
         use_fs = fs if fs else self.default_fs
+        if not self.has_fs(use_fs):
```

Comment:
Can we save call for `has_fs(use_fs)` in case of `default_fs` (because I feel no need to validate `default_fs`) ?